### PR TITLE
Members

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]
Think about the VSCode extension ;)

**Comment by LoicBrassart** [comment]
No validation from the SM, please assign me later!

**Review by Hacene010 — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -23,5 +23,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-edTechHdf.vuerd
 .env
+.vscode
+
+
+
```
Comment (LoicBrassart) [inline .gitignore]: Useless empty lines

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Clean your .gitignore please

**Review by Hacene010 — APPROVED** [review]
all ok.

**Review by LoicBrassart — APPROVED** [review]
NmE!

**Pull request merged**